docs/index.rst

```diff
@@ -2,22 +2,12 @@
 CrateDB DBAL Documentation
 ==========================
 
-Introduction
-============
-
 `DBAL`_ is a database abstraction layer with features for database schema
 introspection, schema management and `PDO`_ abstraction written in `PHP`_.
 
 `crate-dbal`_ is an implementation of this abstraction layer for `CrateDB`_.
 
-Installation
-============
-
-If you are new to CrateDB PDO you might want to start with the
-:doc:`installation` section of the documentation.
-
-Table of Contents
-=================
+.. rubric:: Table of Contents
```

Comment:
Does that list installation and usage? IMHO we should keep the installation reference.

Reply:
yes it does :)